### PR TITLE
Focus Last Element In List On Load More

### PR DESCRIPTION
**Pull request by NicholasHallman**
US129321

Currently, when the user clicks the 'Load More' button nothing is in focus once the button disapears. This is an issue if the user is using a keyboard to navigate as they have to navigate all the way back down the page. 

This PR focuses the last learner in the list so as more learners load in the user will be able to select them efficiently.

### CODE REVIEW

File: components/user-selector.js [hunk 1]
```diff
@@ -290,6 +290,8 @@ class UserSelector extends SkeletonMixin(Localizer(MobxLitElement)) {
 	}
 
 	_onLoadMore() {
+		const buttonItems = Array.from(this.shadowRoot.querySelectorAll('d2l-insights-list-item-button'));
+		buttonItems[buttonItems.length - 1].focus();
```
Comment (bemailloux) [inline components/user-selector.js:294]: What happens if there are no buttonItems?

Comment (NicholasHallman) [reply components/user-selector.js:294]: Good catch!

**Review by bemailloux — APPROVED** [review]

**Review by JikaiLong — APPROVED** [review]

**Comment by NicholasHallman** [comment]
Waiting on #627 before merging

**Comment by JikaiLong** [comment]
> Waiting on #627 before merging
#627 is merged, feel free to merge this one.

**Pull request merged**